src/device.rs

```diff
@@ -343,13 +357,15 @@ pub trait DeviceV1_0 {
 
     unsafe fn cmd_bind_vertex_buffers(&self,
                                       command_buffer: vk::CommandBuffer,
+                                      first_binding: vk::uint32_t,
                                       buffers: &[vk::Buffer],
-                                      offsets: &vk::DeviceSize) {
+                                      offsets: &[vk::DeviceSize]) {
+        debug_assert_eq!(buffers.len(), offsets.len());
         self.fp_v1_0().cmd_bind_vertex_buffers(command_buffer,
-                                               0,
+                                               first_binding,
```

Comment:
Is this necessary? We already use a slice here which already has the information where it should start. For example

`device.cmd_bind_vertex_buffers(cb, &buffers[5 .. 10], &offsets);`